compiler/src/iree/compiler/Dialect/Stream/Transforms/test/encode_host_tensors.mlir

```diff
@@ -50,17 +46,12 @@ util.func public @sizeof_rhs_encoding_dynamic(%arg0: index, %arg1: index) -> ind
   %0 = stream.tensor.sizeof tensor<?x?xf32, #encoding>{%arg0, %arg1} : index
   util.return %0 : index
 }
+// CHECK-DAG:   #[[$MAP:.+]] = affine_map<()[s0, s1] -> (((s1 ceildiv 8) * ((s0 ceildiv 16) * 64)) * 8)>
```

Comment:
This simplifies the computation (e.g., 64=16*4), but it adds overheads for the human. Do we want to use affine.apply ops in this case?